eng/codeOptimization.targets

```diff
@@ -0,0 +1,19 @@
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(ApplyPartialNgenOptimization)' == ''">
+     <ApplyPartialNgenOptimization Condition="'$(IsReferenceAssembly)' == 'true'">false</ApplyPartialNgenOptimization>
+     <ApplyPartialNgenOptimization Condition="'$(GeneratePlatformNotSupportedAssembly)' == 'true' OR '$(GeneratePlatformNotSupportedAssemblyMessage)' != ''">false</ApplyPartialNgenOptimization>
+  </PropertyGroup>
+
+  <Target Name="SetApplyPartialNgenOptimization"
```

Comment:
Just to double-check because the naming makes me suspicious: this is only about embedding the optimization data into the produced assemblies and not about actually running partial NGen on them, correct?

We have a big mail thread going on trying to decide which assemblies would be okay to partially NGen without sacrificing important perf metrics.

Reply:
> Just to double-check because the naming makes me suspicious: this is only about embedding the optimization data into the produced assemblies and not about actually running partial NGen on them, correct?

Correct, this is only about embedding data into the assembly itself.